egs/thchs30/s5/run.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+. ./cmd.sh ## You'll want to change cmd.sh to something that will work on your system.
+           ## This relates to the queue.
+. ./path.sh
+
+H=`pwd`
+
+thchs=/work3/zxw/thchs30/data_thchs30
+# corpus and trans directory
+# you can obtain the database by uncommting the following lines
+# ( cd `dirname $thchs`
+#     echo "downloading THCHS30 at $PWD ..."
+#     wget http://www.openslr.org/resources/18/data_thchs30.tgz
+#     tar xvf data_thchs30.tgz
+# )
+n=8
+
+# data preparation 
+# generate text, wav.scp, utt2pk, spk2utt
+
+
+local/thchs-30_data_prep.sh $H $thchs || exit 1;
+
+#produce MFCC features 
+rm -rf data/mfcc && mkdir -p data/mfcc &&  cp -R data/{train,dev,test,test.ph} data/mfcc || exit 1;
+for x in train dev test; do
+   #make  mfcc 
+   steps/make_mfcc.sh --nj $n --cmd "$train_cmd" data/mfcc/$x exp/make_mfcc/$x mfcc/$x || exit 1;
+   #compute cmvn
+   steps/compute_cmvn_stats.sh data/mfcc/$x exp/mfcc_cmvn/$x mfcc/$x/_cmvn || exit 1;
+done
+#copy feats and cmvn to test.ph, avoid duplicated mfcc & cmvn 
+cp data/mfcc/test/feats.scp data/mfcc/test.ph && cp data/mfcc/test/cmvn.scp data/mfcc/test.ph || exit 1;
```

Comment:
we don't normally put dots in directory names (like ph), and it's not clear what ph means here.